docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -36,6 +36,17 @@ In virtual environments, high CPU steal indicates that the virtual machines migh
 
 *Further Reading*: https://docs.couchbase.com/server/current/install/best-practices-vm.html[Virtualisation Best Practices]
 
+[#CB90043]
+=== Disk Commit Fail (CB90043)
+
+*Background*: A disk commit failure is when Couchbase Server cannot write data from memory to the file system.
+
+*Condition*: An item failed to be committed from memory to disk.
```

Comment:
```suggestion
*Condition*: An item failed to be written to disk.
```

Trying to make this understandable for people without deep knowledge of our innards, while still reasonably technical (since that's the point of the Condition field) - what do you think?